LG-4125 Rules of use (#630)
18F/identity-site: content/_en/policy/rules-of-use.md

---
layout: sidenav
title: "Rules of Use"
description: Rules of use.
permalink: /policy/rules-of-use/
sidenav: policies
---

> # **Rules of Use**

Reviewer: bold inside a header seems redundant?

```suggestion
# Rules of Use
```

The login.gov service employs recognized security and privacy best practices.

Prior to using the login.gov service, you are required to agree to these terms of service. When the login.gov service changes its terms of service, you will be given the option to agree or to decline the updated terms of service. Similarly, when conducting certain activities in the login.gov service, such as providing personal information for the first time, the login.gov service may require you to re-confirm your understanding of how it uses your information.

# 2. Your Agreement

In addition to details about your consent stated elsewhere in these terms, by accepting these terms and using the login.gov service, you agree that:

> 1. Are not a child under 13 years of age,

Reviewer 2: I see it's written this way in the document, but the grammar here seems incorrect.

```suggestion
1. You are not a child under 13 years of age,
```

Not sure who'd need to sign off on such a change though.

# 5. Termination

You can terminate your account at any time through your account profile. Additionally, in the event of fraud or other violations of these Rules of Use, we may revoke access to your account. If this occurs, we will still protect your account information consistent with our [Privacy Policy](https://login.gov/policy/) and System of Record Notice [here](https://www.federalregister.gov/documents/2017/01/19/2017-01174/privacy-act-of-1974-notice-of-a-new-system-of-records) and [here](https://www.federalregister.gov/documents/2017/08/10/2017-16852/privacy-act-of-1974-system-of-records).

In the unlikely event that the login.gov service is discontinued, information on the account will be retained subject to our System of Records Notice, as amended, which you can access [here](https://www.federalregister.gov/documents/2017/01/19/2017-01174/privacy-act-of-1974-notice-of-a-new-system-of-records) and [here](https://www.federalregister.gov/documents/2017/08/10/2017-16852/privacy-act-of-1974-system-of-records).

# 6. Authorities

> The login.gov service is operated within the United States, but is accessible globally for public use. The login.gov service is operated by the General Services Administration under authorities and guidance found in[6 USC § 1523 (b)(1)(A)-(E)](https://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title6-section1523&amp;num=0&amp;edition=prelim), the [E-Government Act of 2002](https://www.congress.gov/107/plaws/publ347/PLAW-107publ347.pdf) ([44 USC § 3501](https://www.govinfo.gov/content/pkg/PLAW-104publ13/html/PLAW-104publ13.htm)), and [40 USC § 501](https://www.gsa.gov/cdnstatic/CNXII__Eligibility_to_Use_GSA.pdf), and [OMB M-19-17](https://www.whitehouse.gov/wp-content/uploads/2019/05/M-19-17.pdf).

Reviewer 2: Minor: Missing a space

```suggestion
The login.gov service is operated within the United States, but is accessible globally for public use. The login.gov service is operated by the General Services Administration under authorities and guidance found in [6 USC § 1523 (b)(1)(A)-(E)](https://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title6-section1523&amp;num=0&amp;edition=prelim), the [E-Government Act of 2002](https://www.congress.gov/107/plaws/publ347/PLAW-107publ347.pdf) ([44 USC § 3501](https://www.govinfo.gov/content/pkg/PLAW-104publ13/html/PLAW-104publ13.htm)), and [40 USC § 501](https://www.gsa.gov/cdnstatic/CNXII__Eligibility_to_Use_GSA.pdf), and [OMB M-19-17](https://www.whitehouse.gov/wp-content/uploads/2019/05/M-19-17.pdf).
```

We will never share information with a partner without your explicit consent. You may revoke this consent at any time, and we will revoke the partner&#39;s access to the information. The partner may retain it subject to their data retention policies. The login.gov service does not collect any information you provide directly to the partner.

Prospective partners and other interested parties can find more information about the partner onboarding process [here](https://developers.login.gov/).

# 9. Fees

There are no fees charged to the user for registering or using the login.gov service. There is no fee associated with connecting to our partners, though the partners themselves may charge fees for some of their services, such as making a reservation, purchasing an item, or applying for a service.

# 10. Representations, Warranties and Liabilities

Disclaimer of Warranties

> Login.gov services are provided &quot;as is&quot; and on an &quot;as-available&quot; basis. GSA hereby disclaims all warranties of any kind, express or implied, including without limitation the warranties of merchantability, fitness for a particular purpose, and non-infringement. GSA makes no warranty that the services will be error free or that access thereto will be continuous or uninterrupted.

Reviewer: let's just use plain quotes since markdown will smartify them for us?

```suggestion
Login.gov services are provided "as is" and on an "as-available" basis. GSA hereby disclaims all warranties of any kind, express or implied, including without limitation the warranties of merchantability, fitness for a particular purpose, and non-infringement. GSA makes no warranty that the services will be error free or that access thereto will be continuous or uninterrupted.
```

## 1. General Representations

You hereby warrant that (1) your use of the website and services will be in strict accordance with the Agreement and all applicable laws and regulations, and (2) your use of the website and services will not infringe or misappropriate the intellectual property rights of any third party.

## 2. Limitations on Liability

In no event will GSA be liable with respect to any subject matter of this Agreement under any contract, negligence, strict liability or other legal or equitable theory for: (1) any special, incidental, or consequential damages; (2) the cost of procurement of substitute products or services; or (3) for interruption of use or loss or corruption of data.

# 11. General Provisions

## 1. Entire Agreement

This Agreement constitutes the entire Agreement between GSA and you concerning the use of the website and services, and may only be modified by the posting of a revised version on this page by GSA.

## 2. Disputes

Any disputes arising out of this Agreement and access to or use of the services shall be governed by federal law.

## 3. No Waiver of Rights

> GSA&#39;s failure to exercise or enforce any right or provision of this Agreement shall not constitute waiver of such right or provision.

Reviewer: ```suggestion
GSA's failure to exercise or enforce any right or provision of this Agreement shall not constitute waiver of such right or provision.
```

let the markdown do the quotes?
